.tekton/scanner-slim-pull-request.yaml

```
@@ -237,12 +245,21 @@ spec:
         workspace: workspace
       - name: basic-auth
         workspace: git-auth
+    - name: determine-image-tag
+      runAfter:
+      # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
+      - clone-repository
+      taskRef:
+        name: determine-image-tag
+      workspaces:
+      - name: source
+        workspace: workspace
     - name: prefetch-dependencies
```

[nit] A whitespace around the task would not hurt.

```suggestion
        workspace: git-auth

    - name: determine-image-tag
      runAfter:
      # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
      - clone-repository
      taskRef:
        name: determine-image-tag
      workspaces:
      - name: source
        workspace: workspace

    - name: prefetch-dependencies
```

---

done - both `scanner-slim` pipelines were inconsistent in the formatting.